README.md

```diff
@@ -18,15 +18,16 @@ This repository interfaces Service Discovery, in this instance CloudMap, in orde
 
 
 ## TODO
-- Lambda (`request`).
-- SQS (`listen`).
 - Http (`request`|`call`).

```

Comment:
IMO this is on a plan to keep it in line with other SDKs. what do you think @EwanValentine ?

Reply:
Agreed! These were under the # TODO heading, but I believe they're done right? Although I guess we're not actually exposing the verbs in this library yet, just `call`. So you're right! I'll add these back in